service/src/main/java/org/cbioportal/service/util/VariantType.java

```diff
@@ -2,12 +2,12 @@
 
 public enum VariantType {
     FUSION("Fusion"),
-    TRUNCATION("Truncation"),
```

Comment:
I really don't think it's a good idea to design VT based on OncoKB. It should be other way around. I would just do a mapping at the place of making oncokb call.

Reply:
That call would be in the frontend. It would be much cleaner if it is here. Also it would be useful for validation while importing data

Reply:
@kalletlak do you want the data in portal map VT to deletion when it's TRUNCATION? Why do we have TRUNCATION in the first place then?

Reply:
this is not yet used. All enums in this are based on the most common types based on the data that is present in public database. This would all go away once all the fusion are moved to structural variant table

Reply:
@jjgao could you comment here about the TRUNCATION? I assume it should be mapped to Deletion, but just want to make sure.

Reply:
I don't quite understand what VariantType is for, but if this will go away once switch to SV table, I am ok with any solution that would work for the moment.

Reply:
@jjgao yes, this goes way. But but we could use this for validation while importing structural variant